RNA-seq/05-nb_cell_line_DESeq2-live.Rmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "Neuroblastoma Cell Line: Differential expression analysis with DESeq2"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2018**
+
+In this notebook, we'll perform an analysis to identify the genes that are 
+differentially expressed in _MYCN_ amplified vs. nonamplified neuroblastoma 
+cell lines. 
+
+These RNA-seq data are from 
+[Harenza, et al. _Scientific Data._ 2017.](https://doi.org/10.1038/sdata.2017.33).
+
+More information about DESeq2 can be found in the [excellent vignette](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) 
+from Love, Anders, and Huber from which this is adapted 
+(see also: [Love, Anders, and Huber. _Genome Biology_. 2014.](https://doi.org/10.1186/s13059-014-0550-8))
+
+DESeq2 takes unnormalized counts or estimated counts and does the following:
+
+* [Estimates size factors](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateSizeFactors)
+* [Estimates dispersion](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/estimateDispersions) 
+* Negative binomial generalized linear model fitting and [Wald statistics](https://www.rdocumentation.org/packages/DESeq2/versions/1.12.3/topics/nbinomWaldTest)
+
+![](diagrams/rna-seq_6.png)
+
+## Libraries and functions
+
+```{r}
+# Load the DESeq2 library
+
```

Comment:
Probably don't need to repeat this again... hopefully they have the idea by now.